### PR TITLE
HBW-198 Finish wrapping task list page together

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ v1.8.0 [unreleased]
 - [#347](https://github.com/latera/homs/pull/347) Improve claim buttons styles.
 - [#349](https://github.com/latera/homs/pull/349) Change colors of priorities.
 - [#346](https://github.com/latera/homs/pull/346) Add code coverage for backend.
+- [#352](https://github.com/latera/homs/pull/352)  Wrap task list page together.
```
Comment (sugarfree1) [inline CHANGELOG.md]: Too many spaces

**Comment by mxsmirnoff** [comment]
Updated

File: hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx [hunk 2]
```diff
@@ -3,22 +3,19 @@ import PropTypes from 'prop-types';
...
 
-  const claimTask = async () => (
-    env.connection.request({
-      url:    `${env.connection.serverURL}/tasks/${task.id}/claim`,
-      method: 'POST'
-    })
-    // Fire event (or call callback) here to change task state in the list
-  );
+  const claimAndClose = () => {
+    claimTask(task);
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx]: `await` for clarity sake

File: hbw/app/javascript/packs/hbw/components/helpers/claiming_tasks.js [hunk 3]
```diff
@@ -62,8 +62,22 @@ export default WrappedComponent => class WithClaimingTasks extends Component {
...
+  claimTask = async (task) => {
+    const { subscription } = this.state;
+    const { connection } = this.props.env;
+
+    await connection.request({
+      url:    `${connection.serverURL}/tasks/${task.id}/claim`,
+      method: 'POST'
+    });
+
+    return subscription.poll();
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/helpers/claiming_tasks.js]: Not sure you need to return value of the next poll. I think if you want to return something from this function you should return result of main request (line 70).
Also I doubt about `claimTask` function should perform poll. Looks like it's not its responsibility.

**Pull request merged**